src/Raven.Client/Documents/Operations/Revisions/RevisionsBinConfiguration.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Raven.Client.Documents.Conventions;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Revisions
+{
+    public sealed class RevisionsBinConfiguration : IDynamicJson
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether the revisions bin cleaner is disabled.
+        /// </summary>
+        /// <value>
+        /// <c>true</c> if the cleaner is disabled; otherwise, <c>false</c>.
+        /// </value>
+        public bool Disabled { get; set; }
+
+        /// <summary>
+        /// Gets or sets the minimum age of revisions to keep in the database.
+        /// The revisions-bin cleaner deletes revisions that are older than that.
```

Comment:
we need to be here explicit about revisions for a delete document, and not just any revision

Reply:
Done